sktime/series_as_features/compose/_pipeline.py

```diff
@@ -35,55 +37,126 @@ class FeatureUnion(_FeatureUnion, _PanelToPanelTransformer):
     transformer_weights : dict, optional
         Multiplicative weights for features per transformer.
         Keys are transformer names, values the weights.
-    preserve_dataframe : bool
-        Save constructed dataframe.
+    preserve_dataframe : bool - deprecated
+    flatten_transform_index : bool, optional (default=True)
+        if True, columns of return DataFrame are flat, by "transformer__variablename"
+        if False, columns are MultiIndex (transformer, variablename)
+        has no effect if return mtypes is one without column names
     """
 
     _required_parameters = ["transformer_list"]
 
+    _tags = {
+        "scitype:transform-input": "Series",
+        "scitype:transform-output": "Series",
+        "scitype:transform-labels": "None",
+        "scitype:instancewise": False, # depends on components
+        "univariate-only": False,  # depends on components
+        "handles-missing-data": False,  # depends on components
+        "X_inner_mtype": ["pd.DataFrame", "pd-multiindex"],
+        "y_inner_mtype": "None",
+        "X-y-must-have-same-index": False,
+        "enforce_index_type": None,
+        "fit-in-transform": False,
+        "transform-returns-same-time-index": False,
+        "skip-inverse-transform": False,
+    }
+
     def __init__(
         self,
         transformer_list,
         n_jobs=None,
         transformer_weights=None,
         preserve_dataframe=True,
```

Comment:
Maybe have a deprecated message? I dont know could get spammy?

Reply:
can you kindly explain for what?
I don't think anything deprecates here? Still fully downwards compatible?

Reply:
In the Parameter comment it has 'preserve_dataframe : bool - deprecated'

Reply:
ah! Yes, indeed, that makes sense.
Or, I should perhaps better think about how to make it downwards compatible.

Reply:
added a deprecation message. The class itself should still be downwards compatible